app/routes/routes_helper.py

```diff
@@ -0,0 +1,16 @@
+from flask import jsonify, abort, make_response
+
+def validate(cls,obj_id):
```

Comment:
This is a good helper, having one to validate the request body would also be good.